### PR TITLE
Docs audit p4 — About

### PR DESCRIPTION
**Pull request by validbeck**
## Internal Notes for Reviewers

> sc-6527

[**LIVE PREVIEW**](https://docs-demo.vm.validmind.ai/pr_previews/beck/sc-6527/docs-audit-p4-about/index.html)

### Centered `figcaption`

| Old | New |
|---|---|
| Screenshot 2024-10-16 at 11 43 56 AM | Screenshot 2024-10-16 at 11 44 12 AM |

### Cleaned up releases

ALL releases got a semi-thorough edit for style, tone, accuracy, and backfill of any relevant docs! 

**EXAMPLE:**

Screenshot 2024-10-16 at 11 49 05 AM

### Single-sourced upgrade instructions

I've now single-sourced the `_how-to-upgrade.qmd` instructions after cleaning them up and added them to both the end of every existing release and the release notes process doc.

| _how-to-upgrade.qmd | Release Notes process |
|---|---|
| Screenshot 2024-10-16 at 11 49 51 AM | Screenshot 2024-10-16 at 11 53 19 AM |

**I'll make a related Story to have the script/notebook add this automatically in the future to reduce manual lift.**

### Other cleanup

Adjusted some other sections to use the more "cosmetically elegant" listing tiles, some special CTAs, etc:

Screenshot 2024-10-16 at 11 54 31 AM
Screenshot 2024-10-16 at 11 54 41 AM
Screenshot 2024-10-16 at 11 54 56 AM
Screenshot 2024-10-16 at 11 55 05 AM

### CODE REVIEW

File: site/releases/2023/2023-aug-15/highlights.qmd
Comment (nrichers) [inline site/releases/2023/2023-aug-15/highlights.qmd:31]: This PR adds **59 of these buttons**. That's a lot of buttons! I'm not opposed to that in principle, but the original intent was to use them very sparingly to make sure they pop. I don't think they are effective at scale ... In which case we might consider switching over to a less loud Tachyons button.

**Review by nrichers — CHANGES_REQUESTED** [review]
I left a couple of comments that we should discuss separately (use of buttons, use of images in listing tiles). Will approve after the 404s are fixed. 

Thank you for this PR! That's more TLC than I ever expected our old release notes to get. 🚀🚀🚀

File: site/releases/2023/2023-aug-15/highlights.qmd
Comment (validbeck) [reply site/releases/2023/2023-aug-15/highlights.qmd:50]: Interesting, they are fine locally, I checked them all. Just for this page? 🤔

File: site/releases/2023/2023-aug-15/highlights.qmd
Comment (validbeck) [reply site/releases/2023/2023-aug-15/highlights.qmd:22]: The bullet points were really squished together for me and harder to parse, and didn't look as good with the button links but you're right that it's still a bit off...

The definition styling doesn't work with the flexboxes but I bolded the headings instead: 

Screenshot 2024-10-17 at 8 31 54 AM

File: site/about/overview-model-documentation.qmd
Comment (validbeck) [reply site/about/overview-model-documentation.qmd:142]: I was torn on whether or not to include the image, but there's already a "special" div at the top of the page for the intro so I wanted to limit that element:

Screenshot 2024-10-17 at 8 48 10 AM

File: site/releases/2023/2023-aug-15/highlights.qmd
Comment (nrichers) [reply site/releases/2023/2023-aug-15/highlights.qmd:50]: > Interesting, they are fine locally, I checked them all. Just for this page? 🤔

This is the only page I checked. The links look like they should work but the preview 404s.

Comment (nrichers) [reply site/releases/2023/2023-aug-15/highlights.qmd:50]: I had this moment of worry that the recent changes to how we handle the Python docs are implicated but they appear fine on staging: https://docs-staging.validmind.ai/validmind/validmind.html. Let me know if you want help troubleshooting?

Comment (validbeck) [reply site/releases/2023/2023-aug-15/highlights.qmd:50]: Hm, I think for some reason the demo-docs doesn't upload the Python docs: https://docs-demo.vm.validmind.ai/validmind/validmind.html

Just to be sure though, I'll just change the URLs to the full one for now.

Comment (validbeck) [reply site/releases/2023/2023-aug-15/highlights.qmd]: I think it's just a weird quirk of how the preview operates — it tries to send people to the top-level `validmind` folder instead of the one that's within the PR preview folder... (even though it doesn't do that for any of the other directories, maybe because the `validmind` exists in output only)?

File: site/releases/2023/2023-aug-15/highlights.qmd
Comment (nrichers) [reply site/releases/2023/2023-aug-15/highlights.qmd]: Yeah, bolding looks better! Much easier to parse — the font size of H5 headings felt messy somehow.

File: site/releases/2023/2023-aug-15/highlights.qmd
Comment (validbeck) [reply site/releases/2023/2023-aug-15/highlights.qmd]: They should be good now but you'll need to empty cache/hard refresh/etc. hehe

Comment (nrichers) [reply site/releases/2023/2023-aug-15/highlights.qmd]: > Hm, I think for some reason the demo-docs doesn't upload the Python docs: https://docs-demo.vm.validmind.ai/validmind/validmind.html

The docs-demo site is quite old, I would not use it for reference. 

(Related, how do you feel about repurposing docs-demo for the chatbot? Once I figure it out with Cullen where the backend should live, we could use docs-demo for further exploration.)

Comment (validbeck) [reply site/releases/2023/2023-aug-15/highlights.qmd]: > Related, how do you feel about repurposing docs-demo for the chatbot? 

Go for it!

Comment (nrichers) [reply site/releases/2023/2023-aug-15/highlights.qmd]: > They should be good now but you'll need to empty cache/hard refresh/etc. hehe

They mostly work now but it should be OK. I think the difference is whether you are using 

`action="https://docs.validmind.ai/validmind/validmind/tests.html#describe_test`

or 

`action="/validmind/validmind.html#get_test_suite`

The latter, relative links seem to 404 but I expect they will work after you merge version. Imma approve. 🚀

**Pull request merged**
